reference_codegen/generate.js

```diff
@@ -178,13 +196,13 @@ Object.entries(helpAll.name_to_target_type_info).forEach(([name, info]) => {
 // `_category_.json` files
 fs.writeFileSync(
   "goals/_category_.json",
-  '{\n  "label": "Goals",\n  "link": {\n    "type": "generated-index",\n    "slug": "/reference/goals",    "title": "Goals"\n  }}\n'
+  '{\n  "label": "Goals",\n  "link": {\n    "type": "generated-index",\n    "slug": "/reference/goals",\n    "title": "Goals"\n  }\n}\n'
 );
 fs.writeFileSync(
   "subsystems/_category_.json",
-  '{\n  "label": "Subsystems",\n  "link": {\n    "type": "generated-index",\n    "slug": "/reference/subsystems",    "title": "Subsystems"\n  }}\n'
+  '{\n  "label": "Subsystems",\n  "link": {\n    "type": "generated-index",\n    "slug": "/reference/subsystems",\n    "title": "Subsystems"\n  }\n}\n'
 );
 fs.writeFileSync(
   "targets/_category_.json",
-  '{\n  "label": "Targets",\n  "link": {\n    "type": "generated-index",\n    "slug": "/reference/targets",\n    "title": "Targets"\n  }}\n'
+  '{\n  "label": "Targets",\n  "link": {\n    "type": "generated-index",\n    "slug": "/reference/targets",\n    "title": "Targets"\n  }\n}\n'
 );
```

Comment:
Just a minor style thing (no change required): We could potentially use template literals (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#multi-line_strings) to have these written out more literally and thus more obvious contents. Either that, or separate JSON file templates that can be copied verbatim.

Reply:
Ah I didn't know JS template literals preserved newlines.

Yeah feel free to PR id +1 that